src/canopy-react-error-boundary.js

```diff
@@ -8,14 +8,23 @@ export default function decorateOptions(opts) {
 
   return function decorateComponent(Comp) {
     return class CanopyReactErrorBoundary extends React.Component {
+      static displayName = `CanopyReactErrorBoundary(${opts.featureName})`
```

Comment:
So that it can identify which CanopyReactErrorBoundary is throwing the error in the warning message. See the screenshot in Conversation tab